src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -203,7 +203,12 @@ PeakDetectionDialog::PeakDetectionDialog(MainWindow* parent) :
         featureOptions->setChecked(false);
         connect(featureOptions, SIGNAL(toggled(bool)), SLOT(featureOptionsClicked()));
 
-        compoundRTWindow->setEnabled(false); //TODO: Sahil - Kiran, Added while merging mainwindow
+        //TODO: Sahil - Kiran, Added while merging mainwindow
```

Comment:
Let's remove these "TODO: ... added while ..." comments wherever we make changes.